README.md

```diff
@@ -51,20 +51,22 @@ And a corresponding playbook as this (`beegfs.yml`):
           meta: "{{ inventory_hostname in groups['cluster_beegfs_mds'] }}"
```

Comment:
The README could do with a little filling out, especially for non-scalar variables. That can be done in a later commit though.